in.co.nandakumar.vara.yml

```diff
@@ -0,0 +1,23 @@
+id: in.co.nandakumar.vara
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: vara
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --filesystem=host
```

Comment:
why do you need host?

Reply:
To help users save the file wherevery they want. Is there another way to do it? I'd spent at least half an hour going through the doc and doing some experiments before adding this hesitantly. I'd be happy to make changes if there is a better way.

Reply:
If your app uses filechooser portal https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.FileChooser.html and opening and saving goes through a filepicker you don't need any access

Reply:
Looks like it is gtk3, so that involves using Gtkfilechoosernative https://docs.gtk.org/gtk3/class.FileChooserNative.html which uses portal by default.

Reply:
Shall we wait for this to be implemented?